moveit_ros/planning/moveit_cpp/src/planning_component.cpp

```diff
@@ -396,27 +374,9 @@ bool PlanningComponent::setGoal(const std::string& goal_state_name)
   return setGoal(goal_state);
 }
 
-bool PlanningComponent::execute(bool blocking)
-{
-  if (!last_plan_solution_)
-  {
-    RCLCPP_ERROR(LOGGER, "There is no successful plan to execute");
-    return false;
-  }
-
-  // TODO(henningkayser): parameterize timestamps if required
-  // trajectory_processing::TimeOptimalTrajectoryGeneration totg;
-  // if (!totg.computeTimeStamps(*last_solution_trajectory_, max_velocity_scaling_factor_,
-  // max_acceleration_scaling_factor_))
-  //{
-  //  RCLCPP_ERROR("Failed to parameterize trajectory");
-  //  return false;
-  //}
-  return moveit_cpp_->execute(last_plan_solution_.trajectory_, blocking);
-}
-
-const planning_interface::MotionPlanResponse& PlanningComponent::getLastMotionPlanResponse()
+bool PlanningComponent::execute(bool /*blocking*/)
```

Comment:
we should really drop the blocking flag altogether.

Reply:
The "simultaneous execution" PR to ROS2 eventually (I hope so!). It looks like it does use a blocking flag in MoveItCpp but not the blocking flag in PlanningComponent. So I agree, the `blocking` flag here could probably be dropped.

https://github.com/ros-planning/moveit/pull/3243/files